outplacement_final_report_ipf_client/models/client_config.py

```diff
@@ -229,7 +229,7 @@ def post_request(self, outplacement):
             "unikt_id": outplacement.uniq_ref,
             "inskickad_datum": str(outplacement.jp_sent_date),
             "rapportering_datum": outplacement.report_date, 
-            "status": outplacement.status,
+            "status": outplacement.stage_id.sequence,
```

Comment:
Is it sequence number?

Reply:
I think but the number is the number I need